pkg/jobrunaggregator/cmd.go

```diff
@@ -0,0 +1,114 @@
+package jobrunaggregator
+
+import (
+	"bytes"
+	"context"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"google.golang.org/api/option"
+	"gopkg.in/yaml.v2"
+
+	"cloud.google.com/go/storage"
+)
+
+const (
+	openshiftCIBucket string = "origin-ci-test"
```

Comment:
you probably don't care but there are a couple and we could read config to find them